### PR TITLE
txl 10.6d

### PR DESCRIPTION
**Pull request by kai5263499**
This is a New Formula. Txl is a unique programming language "designed to support computer software analysis and source transformation tasks"

### CODE REVIEW

File: txl.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Txl < Formula
+  desc "Expierimental rule-based structural programming language"
+  homepage "http://txl.ca/"
+  url "http://txl.ca/ndownload/26466-txl10.6d.macosx64.tar.gz"
```
Comment (mistydemeo) [inline txl.rb]: Mind adding the 32-bit download too? e.g. in an `if Hardware::CPU.is_64_bit?` block.

**Comment by kai5263499** [comment]
I've updated this formula to include 32bit support per @mistydemeo 's recommendation. I also fixed the whitespace issues.

**Comment by dunn** [comment]
It looks like there are terms and conditions that have to be agreed to in order to use the software? http://txl.ca/ndownload.html

If that's right, please add a caveats section like [`qt5.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/qt5.rb#L160-L161) does.

**Comment by kai5263499** [comment]
That's right @dunn I added a caveats section noting that the terms and conditions were pre-agreed to

**Comment by dunn** [comment]
Thank you for the contribution! Merged in 9db45bd346b05a7b16c799ad259a66429de279d9.

**Pull request closed without merging**